Wire/ValueSerializers/SessionAwareByteArrayRequiringValueSerializer.cs.orig

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.IO;
+using System.Linq.Expressions;
+using System.Reflection;
+using Wire.Compilation;
+
+namespace Wire.ValueSerializers
+{
+    public abstract class SessionAwareByteArrayRequiringValueSerializer<TElementType> : ValueSerializer
+    {
+        private readonly byte _manifest;
+        private readonly MethodInfo _write;
+        private readonly Action<Stream, object, byte[]> _writeCompiled;
+        private readonly MethodInfo _read;
+        private readonly Func<Stream, byte[], TElementType> _readCompiled;
+
+        protected SessionAwareByteArrayRequiringValueSerializer(byte manifest,
+            Expression<Func<Action<Stream, TElementType, byte[]>>> writeStaticMethod,
+            Expression<Func<Func<Stream, byte[], TElementType>>> readStaticMethod)
+        {
+            _manifest = manifest;
+            _write = GetStatic(writeStaticMethod, typeof(void));
+            _read = GetStatic(readStaticMethod, typeof(TElementType));
+
+<<<<<<< HEAD:Wire/ValueSerializers/SessionAwareValueSerializer.cs
```

Comment:
ah, one of the ".cs.orig" files got committed. the merge is done and works, Ill remove the junk file.
Thanks